website/docs/r/synthetics_monitor_step.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_step_monitor"
+sidebar_current: "docs-newrelic-resource-synthetics-step-monitor"
+description: |-
+Create and manage a Synthetics monitor in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_step\_monitor
+
+Use this resource to create, update, and delete the synthetics step monitor in New Relic.
+
+-> **NOTE:** The [newrelic_synthetics_private_location](newrelic_synthetics_private_location.html) resource private minion can take upto 10 minutes to be available through Terraform.
+
+## Example Usage
+
+##### Type: `STEP MONITOR`
+```hcl
+resource "newrelic_synthetics_step_monitor" "foo" {
+  name = "foo"
+  enable_screenshot_on_failure_and_script = true
+  locations_public = ["US_EAST_1", "US_EAST_2"]
+  period = "EVERY_6_HOURS"
+  status = "ENABLED"
+  steps {
+    ordinal = " "
+    types = " "
+    values = "ASSERT_ELEMENT"
+  }
+  tag {
+    key = "some_key"
+    values = ["some_value"]
+  }
+}
+```
+See additional [examples](#additional-examples).
+
+```hcl
+resource "newrelic_synthetics_private_location" "bar1" {
+  description               = "Test Description"
+  name                      = "%[1]A"
+  verified_script_execution = true
+}
+  resource "newrelic_synthetics_step_monitor" "bar" {
+    name             = "step_monitor"
+    uri              = "https://www.one.example.com"
+    locations_public = ["AP_SOUTH_1"]
+    period           = "EVERY_6_HOURS"
+    status           = "ENABLED"
+    tag {
+      key    = "some_key"
+      values = ["some_value"]
+    }
+  }
+```
+## Argument Reference
+
+The following are the common arguments supported for `STEP` monitor:
+
+* `account_id`- (Required) The account in which the Synthetics monitor will be created.
+* `name` - (Required) The name for the monitor.
+* `uri` - (Required) The uri the monitor runs against.
+* `locations_public` - (Required) The location the monitor will run from. Valid public locations are https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips/
```

Comment:
Can we make a note that either only one of `locations_public` or `locations_private` is required?